### PR TITLE
Add scrollTop to position if container is not body

### PR DESCRIPTION
**Pull request by thetimbanks**
This goes with issue #1616 

Original problem: http://jsfiddle.net/thetimbanks/L24bnfpv/
Fixed: http://jsfiddle.net/thetimbanks/L24bnfpv/1/

### CODE REVIEW

**Comment by acrobat** [comment]
Thanks, we will review and merge this PR after the 1.5.0 release!

**Comment by hebbet** [comment]
LGTM

File: dist/js/bootstrap-datepicker.js [hunk 1]
```diff
@@ -677,6 +677,10 @@
 			var left = offset.left - appendOffset.left,
 				top = offset.top - appendOffset.top;
 
+			if (this.o.container !== 'body') {
```
Comment (acrobat) [inline dist/js/bootstrap-datepicker.js]: please revert the change in the dist file, we generate them regular!

**Comment by acrobat** [comment]
@vsn4ik what do you think about this change? (after your fixes in #1571)

I will test the pr myself later!

**Comment by thetimbanks** [comment]
@acrobat Removed the change from the dist js file

**Comment by vsn4ik** [comment]
LGTM

**Comment by acrobat** [comment]
@thetimbanks can you provide a jsfiddle to display the original problem?

**Comment by thetimbanks** [comment]
@acrobat Sure. 

Original problem: http://jsfiddle.net/thetimbanks/L24bnfpv/
Fixed: http://jsfiddle.net/thetimbanks/L24bnfpv/1/

**Comment by acrobat** [comment]
Thanks! I will merge this later this week!

**Comment by acrobat** [comment]
Thanks @thetimbanks!
